src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -100,7 +99,7 @@ public class InternalRequestsController : ResourceControllerBase
                 r.AlwaysAccessWhen().FullControl().FullControlInternal();
                 r.AnyOf(or =>
                 {
-
+                    or.BeResourceOwner(departmentPath, includeParents: false, includeDescendants: true);
```

Comment:
Should maybe also validate that the relevant resource on the instance exists in the department?